linera-service/src/storage.rs

```diff
@@ -123,19 +125,28 @@ impl FromStr for StorageConfig {
         if let Some(s) = input.strip_prefix(SCYLLA_DB) {
             let mut uri: Option<String> = None;
             let mut table_name: Option<String> = None;
+            let parse_error: &'static str = "Correct format is (https|http)://db_hostname:port.";
```

Comment:
Actually maybe here it should be "tcp"? What do you think?

Reply:
Good point!